v1.0-known-limitations.md

```diff
@@ -0,0 +1,38 @@
+---
+title: v1.0 Known Limitations
+summary:
+toc: false
+---
+
+This page identifies limitations we've identified in the v1.0 release and plan to investigate and possibly resolve in a future release.
+
+<div id="toc"></div>
+
+## `INSERT ON CONFLICT` vs. `UPSERT`
```

Comment:
@danhhz, the following is based on https://github.com/cockroachdb/cockroach/issues/14482. We've documented this elsewhere already, but nonetheless, let me know if anything needs tweaking here.